aws/sqs/sqs.go

```diff
@@ -81,7 +81,7 @@ func (s *SQS) Receive(queueURL string, visibilityTimeout int32) (Raw, error) {
 		VisibilityTimeout:   visibilityTimeout,
 		WaitTimeSeconds:     20,
 	}
-	return s.receiveMessage(&input)
+	return s.receiveMessage(&input, context.TODO())
```

Comment:
Simply call `ReceiveWithContext(queueURL, visibilityTimeout)` so we know we're just wrapping context.TODO